openedx/features/data_extract/management/commands/get_course_analytics.py

```diff
@@ -49,17 +51,14 @@ def handle(self, **options):
                     'progress_data': progress_data,
                 })
 
-            with tempfile.TemporaryFile() as tmp:
+            with tempfile.NamedTemporaryFile() as tmp:
+                pyminizip.compress(tmp.name, '/tmp/data.zip', 'philu2miriam', 1)
```

Comment:
for now its ok, you should take password from user in future

Reply:
will do